content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
+
+Channel parameters allow a client to set the properties of a channel. A set of channel parameters is a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
```

Comment:
I think this could do with some further explanation.  Sure channel params allow you to set the properties of a channel, but what does that mean?  This statement simply confirms that channel params are parameters of a channel.  

Let's start with why do we support channel params, why does the reader care, what can they use them for etc.

Reply:
@tomczoink would you be able to provide the context on why we support channel parameters? As far as I can see the how and why comes down to the parameters themselves which we do talk about later on.

Reply:
We provide channel parameters as there are some options we provide on channels that users may find useful. Said options as you say are listed further down the page. I guess could be good to just state something like 'for example, you can specify you want the channel to receive messages as "deltas":LINK, or state a subscription to a channel should get some messages from history as part of subscribing with "rewind":LINK.'

Reply:
addressing in update